src/app/archive_hardfork_toolbox/tests/test_convert_canonical.ml

```diff
@@ -281,57 +249,40 @@ end
 (* Alcotest integration *)
 let test_convert_scenario conn_str
     (name, blocks, expected, target_hash, protocol_version) () =
-  let open Deferred.Let_syntax in
+  let open Deferred.Or_error.Let_syntax in
   (* Create test database *)
   let db_name = sprintf "test_%s" name in
-  let%bind _drop =
-    TestDb.drop_database_if_exists conn_str db_name |> Deferred.ok
-  in
-  let%bind _create = TestDb.create_database conn_str db_name |> Deferred.ok in
-  let%bind _setup = TestDb.create_test_schema conn_str db_name |> Deferred.ok in
-  let%bind _insert =
-    TestDb.insert_blocks conn_str db_name blocks |> Deferred.ok
-  in
+  let%bind () = TestDb.drop_database_if_exists conn_str db_name in
```

Comment:
These are not nit: it's a bug to ignore the results on db exec. Although very likely it'll be catched in the final clause, let's not rely on that behavior and fail fast.